src/components/views/login/InteractiveAuthEntryComponents.js

```diff
@@ -86,12 +89,20 @@ export const PasswordAuthEntry = React.createClass({
             <div>
                 <p>To continue, please enter your password.</p>
                 <p>Password:</p>
-                <input
-                    ref="passwordField"
-                    className={passwordBoxClass}
-                    onChange={this._onPasswordFieldChange}
-                    type="password"
-                />
+                <form onSubmit={this._onSubmit}>
```

Comment:
I've put this in a form which makes the behaviour more semantic and accessible, (we then get submit-on-enter for free, etc)